test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -2193,6 +2193,174 @@ Describe "Invoke-WebRequest tests" -Tags "Feature", "RequireAdminOnWindows" {
             $pathologicalRatio | Should -BeGreaterThan 5
         }
     }
+
+    Context 'WebSession persistence tests' {
+        $connWarning = 'WebSession properties changed: new Http connection will be required'
+
+        It 'Connection persistence maintained' {
+            $uri = Get-WebListenerUrl
+            $null = Invoke-WebRequest -Uri $uri -SessionVariable Session -WarningVariable warnings
+            $warnings | Should -BeNullOrEmpty
+
+            $null = Invoke-WebRequest -Uri $uri -WebSession $Session -WarningVariable warnings
+            $warnings | Should -BeNullOrEmpty
+
+            $null = Invoke-WebRequest -Uri $uri -WebSession $Session -WarningVariable warnings
+            $warnings | Should -BeNullOrEmpty
+
+            $null = Invoke-WebRequest -Uri $uri -WebSession $Session -WarningVariable warnings
+            $warnings | Should -BeNullOrEmpty
+        }
+
+        It 'Connection persistence outperforms non-persistence' {
+            $uri = Get-WebListenerUrl -Https
+            $null = Invoke-WebRequest -Uri $uri -SessionVariable Session -WarningVariable warnings -SkipCertificateCheck
+            $iters = 5
+            $persistence = Measure-Command { 1 .. $iters | ForEach-Object {
+                    $null = Invoke-WebRequest -Uri $uri -WebSession $Session -SkipCertificateCheck
```

Comment:
Indentation should be 4. Below too.

Reply:
Have moved the 1 .. $iters down a line so indentation is correct